docs/sources/panels/transformations.md

```diff
@@ -0,0 +1,193 @@
+
+## Prerequisites
+
+Before you apply transformations, all of the following must be true:
+
+- You have entered a query and returned data from a data source. For more information about queries, refer to [Queries]({{< relref "queries.md" >}}).
+- You have applied a visualization that supports queries, such as:
+  - Graph
+  - Stat
+  - Gauge
+  - Bar gauge
+  - Table
+  - Heatmap
+  - Logs
+
+## Apply a transformation
+
+Transformations are available from the Transform tab in the bottom pane of the Panel Editor, next to the Queries tab.
+
+1. Navigate to the panel that you want to add transformations, click the panel title and then click **Edit**.
+1. Click the **Transform** tab.
+1. Click a transformation to select it. 
+  
+   A transformation row appears that allows you to configure the transformation options.
+
+   Click **Add transformation** to apply another transformation. Keep in mind that the next transformation acts on the result set returned by the previous transformation.
+
+   If you have trouble, click the bug icon to [debug your transformations](#debug-transformations).
+
+   Click the trash can icon to permanently remove a transformation.
+
+{{< docs-imagebox img="/img/docs/transformations/transformations-7-0.png" class="docs-image--no-shadow" max-width= "1100px" >}}
+
+## Transformation types and options
+
+Grafana comes with the following transformations:
+
+- [Reduce](#reduce) - Reduce all rows or data points to a single value using a function like max, min, mean, or last.
+- Filter by name - Filter a result set’s fields by name. This might be useful when you want to show only part of your result set.
+- Filter by query - Filter a result set by the refId of the query. This might be useful when your result set consists of multiple time series and you want to show only some of them.
+- Organize fields - Order, filter, and rename the fields in a result set. This transformation is useful when your result set contains for instance non human-readable field names or when - you want to display a table and alter the order of the columns. 
+- Join by field - Join multiple time series from a result set by field.
+- Add field from calculation - Create new fields that are the result of result set row calculation.
+- Labels to fields - Group a series by time and return labels as fields.
+
+Keep reading for detailed descriptions of each type of transformation and the options available for each, as well as suggestions for how to use them.
+
+### Reduce
+
+Apply a _Reduce_ transformation when you want to simplify your results down to one value. Reduce basically removes time component. If visualized as a table, it reduces a column down to one row (value).
+
+In the **Calculations** field, enter one or more calculation types. Click to see a list of calculation choices. For information about available calculations, refer to the [Calculation list]({{< relref "calculations-list.md" >}}).
```

Comment:
```suggestion
In the **Calculations** field, select one or more calculation types. Click to see a list of calculation choices. For information about available calculations, refer to the [Calculation list]({{< relref "calculations-list.md" >}}).
```

Reply:
"enter" is correct, because you can type them or select.